lib/src/features/course/presentation/course_populated.dart

```diff
@@ -0,0 +1,50 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:the_monkslab_web/src/features/section/_index.dart';
+import 'package:the_monkslab_web/src/models/_index.dart';
+import 'package:the_monkslab_web/src/ui/_index.dart';
+
+class CoursePopulated extends StatelessWidget {
```

Comment:
Well... I should start stop doing this...

Reply:
hehe